plutus-playground-server/src/Playground/Server.hs

```diff
@@ -27,30 +27,32 @@ import           Servant.API                  ((:<|>) ((:<|>)))
 import           Servant.Server               (Handler, Server)
 import           Wallet.Rollup                (doAnnotateBlockchain)
 
-acceptSourceCode ::
+compileSourceCode ::
        SourceCode
     -> Handler (Either InterpreterError (InterpreterResult CompilationResult))
-acceptSourceCode sourceCode = do
-    let maxInterpretationTime :: Microsecond =
-            fromMicroseconds (80 * 1000 * 1000)
+compileSourceCode sourceCode = do
+    let maxInterpretationTime :: Microsecond
+        maxInterpretationTime = fromMicroseconds (80 * 1000 * 1000)
     r <- liftIO . runExceptT $ PI.compile maxInterpretationTime sourceCode
     case r of
         Right vs -> pure . Right $ vs
         Left (CompilationErrors errors) ->
             pure . Left $ CompilationErrors errors
         Left e -> throwError $ err400 {errBody = BSL.pack . show $ e}
 
-runFunction :: Evaluation -> Handler (Either PlaygroundError EvaluationResult)
-runFunction evaluation = do
-    let maxInterpretationTime :: Microsecond =
-            fromMicroseconds (80 * 1000 * 1000)
+evaluateSimulation ::
+       Evaluation -> Handler (Either PlaygroundError EvaluationResult)
+evaluateSimulation evaluation = do
+    let maxInterpretationTime :: Microsecond
```

Comment:
It's the same as l.34/35 so it should just be a top-level definition

Reply:
Agreed. Thanks.